source/async_stack.cpp

```diff
@@ -39,10 +42,47 @@ extern "C" {
 inline pthread_key_t folly_async_stack_root_tls_key = 0xFFFF'FFFFu;
 }
 
-#endif  // FOLLY_ASYNC_STACK_ROOT_USE_PTHREAD
+#else
+
+#include <vector>
+
+// TODO: expose array of AsyncStackRootHolder
```

Comment:
This is a todo because we need to access this out of process. In which case, we may need a tls key after all